src/pages/[userId]/prospects.js

```diff
@@ -1,54 +1,157 @@
 import { useRouter } from 'next/router';
+import styled from '@emotion/styled';
 import Grid from '../../components/utils/Grid';
+import { DesktopOnly, MobileOnly } from '../../components/MediaQuery';
 
 export default function Prospects() {
   const router = useRouter();
   const { userId } = router.query;
 
+  const Container = styled.div`
```

Comment:
There's some weird formatting going on in this component definition.

Reply:
Do you mean the inconsistent spacing? If so, I fixed it. If not, could you be more specific?

Reply:
Also, the media query on lines 54-77 is not indented correctly.

Reply:
I totally missed that. Should be fixed. Let me know if you want it formatted a different way.